psi-ms.obo

```diff
@@ -20081,6 +20081,42 @@ name: LTQ Orbitrap Velos Pro
 def: "Thermo Scientific LTQ Orbitrap Velos Pro, often just referred to as the Orbitrap Velos Pro." [PSI:MS]
 is_a: MS:1000494 ! Thermo Scientific instrument model
 
+[Term]
+id: MS:1003097
+name: MaxQuant protein group-level score
+def: "The probability based MaxQuant protein group score." [PSI:MS]
+is_a: MS:1002368 ! search engine specific score for protein groups
+
+[Term]
+id: MS:1003098
+name: Andromeda peptide PEP
+def: "Peptide probability from Andromeda." [PSI:MS]
+is_a: MS:1002358 ! search engine specific peptide sequence-level identification statistic
+
+[Term]
+id: MS:1003099
+name: MaxQuant-DIA peptide PEP
+def: "Peptide probability from MaxQuant-DIA algorithm." [PSI:MS]
+is_a: MS:1002358 ! search engine specific peptide sequence-level identification statistic
+
+[Term]
+id: MS:1003100
+name: MaxQuant-DIA score
+def: "PSM evidence score from MAxQuant-DIA algorithm." [PSI:MS]
+is_a: MS:1001143 ! PSM-level search engine specific statistic
+
+[Term]
+id: MS:1003101
+name: MaxQuant-DIA PEP
+def: "PSM evidence PEP probability from MAxQuant-DIA algorithm." [PSI:MS]
+is_a: MS:1001143 ! PSM-level search engine specific statistic
+
+[Term]
+id: MS:1003102
+name: NIST msp comment
+def: "NIST msp comment" [PSI:MS]
```

Comment:
What is a NIST msp comment??

Reply:
In the NIST msp file format, one of the field of the spectrum is Comment. We need a Cvterm for it. See example here: https://raw.githubusercontent.com/HUPO-PSI/mzSpecLib/master/legacy-formats/msp-files/nist-bsa_consensus.msp